presto-docs/src/main/sphinx/release/release-0.199.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.199
+=============
+
+General Changes
+---------------
+
+* Add support for spatial LEFT join.
+* Add :func:`hmac_md5`, :func:`hmac_sha1`, :func:`hmac_sha256`, and :func:`hmac_sha512` functions.
+* Add :func:`array_sort` function that takes a lambda as a comparator.
+* Add :func:`line_locate_point` geospatial function.
+* Add support for authentication with JWT access token.
+* Add support for ``ORDER BY`` clause in aggregations for queries that use grouping sets.
+* Expand grouped execution support to ``GROUP BY`` and ``UNION ALL``, making it possible
+  to execute aggregations with less peak memory usage.
+* Change the signature of ``round(x, d)`` and ``truncate(x, d)`` functions so that
+  ``d`` is of type ``INTEGER``. Previously, ``d`` could be of type ``BIGINT``.
+  This behavior can be reverted with the ``deprecated.legacy-round-n-bigint`` config option
+  or the ``legacy_round_n_bigint`` session property.
+* Add support for installing Presto server RPM to machines with OpenJDK.
```

Comment:
I'm thinking this should also be a separate section.

Reply:
Any ideas for the title of that section?

Reply:
How about “Server RPM Changes”?

Then remove the now-redundant words: “Support installing on machines with OpenJDK. Previously, only Oracle JDK was supported.”